src/exec.js

```diff
@@ -24,7 +24,7 @@ function execSyncRead(command, silent = false) {
   if (!silent) {
     console.log(normalized);
   }
-  return _.trim(String(cp.execSync(normalized, { stdio: ['pipe', 'pipe', 'pipe'] })));
+  return _.trim(String(cp.execSync(normalized, { stdio: ['inherit', 'pipe', silent ? 'pipe' : 'inherit'] })));
```

Comment:
Can't we just listen to outputs sent on the pipe between the child and parent processes?

Reply:
(`child.stderr`)

Reply:
ahh what's the point in this, bad idea Asaf